corehq/apps/userreports/tasks.py

```diff
@@ -289,34 +287,6 @@ def delete_data_source_task(domain, config_id):
     delete_data_source_shared(domain, config_id)
 
 
-@periodic_task(run_every=crontab(minute='*/5'), queue=settings.CELERY_PERIODIC_QUEUE)
-def _reprocess_archive_stubs():
-    reprocess_archive_stubs.delay()
```

Comment:
I think that this doesn't matter since its a periodic task and missed tasks should matter, but flagging that any tasks currently in the queue will be dropped because this changes the tasks function signature

Reply:
Good point, and I agree it won't matter in this case because the tasks the way we're using them don't represent individual jobs; rather as a stream they represent the need to continually do a single task regularly.